protocol/contracts/beanstalk/AppStorage.sol

```diff
@@ -510,33 +418,21 @@ contract Storage {
  * @title AppStorage
  * @author Publius
  * @notice Defines the state object for Beanstalk.
- * @param deprecated_index DEPRECATED: Was the index of the BEAN token in the BEAN:ETH Uniswap V2 pool.
- * @param deprecated_cases DEPRECATED: The 24 Weather cases used in cases V1 (array has 32 items, but caseId = 3 (mod 4) are not cases)
  * @param paused True if Beanstalk is Paused.
  * @param pausedAt The timestamp at which Beanstalk was last paused.
  * @param season Storage.Season
- * @param c Storage.Contracts
- * @param f Storage.Field
- * @param g Storage.Governance
+ * @param field Storage.Field
  * @param co Storage.CurveMetapoolOracle
- * @param r Storage.Rain
- * @param s Storage.Silo
+ * @param rain Storage.Rain
+ * @param silo Storage.Silo
  * @param reentrantStatus An intra-transaction state variable to protect against reentrance.
- * @param w Storage.Weather
+ * @param weather Storage.Weather
  * @param earnedBeans The number of Beans distributed to the Silo that have not yet been Deposited as a result of the Earn function being called.
- * @param deprecated DEPRECATED - 14 slots that used to store state variables which have been deprecated through various updates. Storage slots can be left alone or reused.
  * @param a mapping (address => Account.State)
- * @param deprecated_bip0Start DEPRECATED - bip0Start was used to aid in a migration that occured alongside BIP-0.
- * @param deprecated_hotFix3Start DEPRECATED - hotFix3Start was used to aid in a migration that occured alongside HOTFIX-3.
- * @param fundraisers A mapping from Fundraiser ID to Storage.Fundraiser.
- * @param fundraiserIndex The number of Fundraisers that have occured.
- * @param deprecated_isBudget DEPRECATED - Budget Facet was removed in BIP-14.
  * @param podListings A mapping from Plot Index to the hash of the Pod Listing.
  * @param podOrders A mapping from the hash of a Pod Order to the amount of Pods that the Pod Order is still willing to buy.
  * @param siloBalances A mapping from Token address to Silo Balance storage (amount deposited and withdrawn).
  * @param ss A mapping from Token address to Silo Settings for each Whitelisted Token. If a non-zero storage exists, a Token is whitelisted.
```

Comment:
rename to `SiloSettings`

Reply:
fixed in 7ea86b8f732de1bd4bea2313c197882e15f888b6